zerver/lib/webhooks/git.py

```diff
@@ -86,6 +88,10 @@ def get_pull_request_event_message(
         main_message += '\n' + CONTENT_MESSAGE_TEMPLATE.format(message=message)
     return main_message.rstrip()
 
+def get_ping_message(user_name, integration):
```

Comment:
I was thinking of a message like `{integration} webhook has been successfully configured` with an optional `by {user_name}` appended to it, if the username exists.